### PR TITLE
Add title to artifact collection per VM task

### PR DESCRIPTION
**Pull request by mykaul**

### CODE REVIEW

File: contrib/ovirt/lib/ovirtlago/__init__.py [hunk 1]
```diff
@@ -524,7 +524,7 @@ def collect_artifacts(self, output_dir):
         os.makedirs(output_dir)
 
         def _collect_artifacts(vm):
-            with LogTask('%s' % vm.name()):
+            with LogTask('Collecting artifacts from VM %s' % vm.name()):
```
Comment (david-caro) [inline contrib/ovirt/lib/ovirtlago/__init__.py]: For the tasks use base form of the verb instead, gives the same info with less words and makes a little more sense talking about tasks:

  Collect artifacts from VM %s

**Comment by david-caro** [comment]
you will have to rebase, as a guideline:

    git fetch lago-project
    git rebase lago-project/master
    git push mykaul collect_arts --force

File: contrib/ovirt/lib/ovirtlago/__init__.py (hunk 1 above)
Comment (mykaul) [inline contrib/ovirt/lib/ovirtlago/__init__.py]: Perhaps, but in English 'Collect artifacts from ..' sounds wrong.

Comment (david-caro) [inline contrib/ovirt/lib/ovirtlago/__init__.py]: I see no wrong in that phrase, why does it sound wrong to you?

David Caro

El 28/2/2016 12:29, Yaniv Kaul <notifications@github.com> escribió:

In contrib/ovirt/lib/ovirtlago/__init__.py:

> @@ -524,7 +524,7 @@ def collect_artifacts(self, output_dir): > os.makedirs(output_dir) > > def _collect_artifacts(vm): > - with LogTask('%s' % vm.name()): > + with LogTask('Collecting artifacts from VM %s' % vm.name()): 

Perhaps, but in English 'Collect artifacts from ..' sounds wrong.

**Pull request closed without merging**